rasa/nlu/test.py

```diff
@@ -1050,12 +1051,24 @@ def get_eval_data(
                 response_prediction_key, {}
             ).get(OPEN_UTTERANCE_PREDICTION_KEY, {})
 
+            response_prediction_full_intent = selector_properties.get(
+                response_prediction_key, {}
+            ).get("full_retrieval_intent", {})
+
+            if isinstance(response_prediction_full_intent, str):
+                response_prediction_full_intent = response_prediction_full_intent.split(
+                    "/"
+                )[1]
```

Comment:
I would suggest not splitting the predicted `retrieval_intent/sub_intent` because you could have same `sub_intent` under multiple retrieval intents. So, let's compare for e.g. `faq/ask_name` to `faq/ask_weather` and not `ask_name` to `ask_weather`.